.github/workflows/continous-integration.yml

```diff
@@ -285,6 +286,136 @@ jobs:
         COVERALLS_SERVICE_NAME: github
       run: poetry run coveralls
 
+  integration_test:
+    name: Run Integration Tests
+    runs-on: ubuntu-latest
+    needs: [changes]
+    env:
+      REDIS_HOST: localhost
+      REDIS_PORT: 6379
+      POSTGRES_HOST: localhost
+      POSTGRES_PORT: 5432
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      RABBITMQ_HOST: localhost
+      RABBITMQ_PORT: 5672
+      RABBITMQ_USER: guest
+      RABBITMQ_PASSWORD: guest
+
+    services:
+      redis:
+        image: redis
```

Comment:
wondering if we should pin the version as using `latest` is against best practices. What do you think?

Reply:
yes that's better. Wondering if we should pin the exact version or only the major. I'd go with the major (Redis seems to update major tags with the latest micro). WDYT?

Reply:
Sounds good 👍🏻 We are not using any crazy feature so functionality shouldn't break too quickly anyway

Reply:
I can't pin rabbitmq's version because I'm using a version with custom healthchecks and this repo doesn't push tags 🤦🏻 . I think the right thing to do here is to fork the repo and introduce versions (or ask them to push versioned tags instead of just `latest`). But I don't think I can do this in a reasonable timeframe 😖